ldms/src/contrib/sampler/variorum_sampler/Makefile.am

```diff
@@ -10,7 +10,7 @@ COMMON_LIBADD = -lsampler_base -lldms -lovis_util -lcoll \
 		@LDFLAGS_GETTIME@
 
 libvariorum_sampler_la_SOURCES = variorum_sampler.c
-libvariorum_sampler_la_LIBADD = $(COMMON_LIBADD) -lvariorum -ljansson
+libvariorum_sampler_la_LIBADD = $(COMMON_LIBADD) $(LTLIBVARIORUM) $(LTLIBJANSSON) -lvariorum -ljansson
```

Comment:
The LT* variables already contain the "-lvariorum" and "-ljansson" options, so they can be removed from this line.